bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1625,7 +1627,8 @@ use_crop=".true.">lnd/clm2/surfdata_map/ctsm5.1.dev052/landuse.timeseries_mpasa1
 <snicar_solarspec    >mid_latitude_winter</snicar_solarspec>
 <snicar_dust_optics  >sahara</snicar_dust_optics>
 <snicar_snodst_intmix>.false.</snicar_snodst_intmix>
```

Comment:
as above, with defining particular physics versions, instead of a default.

Reply:
Again, this follows the Meier example.

Reply:
That's fine with me if it's consistent with how we're defining defaults elsewhere?